### DISCUSSION TITLE
Make labels easier accessible

### DISCUSSION THREAD

**Discussion by CuzImBisonratte** [community/community#15722] [Mobile] [answered]
Would like to have an easier way to add/remove labels on mobile, like on the bottom of the issue page

**Comment by stevepopovich** [comment, marked as answer]
Hi, thanks for the feedback. Like you said, we have the ability to edit labels in an issue and PRs. What exactly are you looking for?

**Reply by CuzImBisonratte** [reply]
I think just that it is a little confusing for people, where they can find them (and also all other issue-settings like asignees,...) 

For example: I needed until this day that i realised that the "i" in the bottom right corner is the button to get to this. 

I think there there is just one thing that could be done, to prevent this:
Change the icon from the "info-icon" to something like an settings-gear to make clearer that this will open the issue-settings

**Reply by stevepopovich** [reply]
Gotcha, fair feedback! I will pass it on to the team for consideration!

**Reply by CuzImBisonratte** [reply]
Thanks, even if this is a small thing i just wanted to give feedback bout it :D
